### ISSUE TITLE
Printf.parse() removed with Julia #32859

### ISSUE THREAD

**Issue by jmkuhn** [JuliaIO/Formatting.jl#87]
Printf.parse() removed with Julia [#32859](https://github.com/JuliaLang/julia/pull/32859)
```
julia> sprintf1("%10.4f", 1.0)
ERROR: MethodError: no method matching parse(::String)
```

**Comment by ronisbr** [comment]
Yes, my package that depends on Formatting is falling in testing with Julia nightly. Is there any workaround?

**Comment by jmkuhn** [comment]
I'll try to get a workaround up in the next day or 2.

**Comment by ScottPJones** [comment]
My package `Strs.jl` also broke (at depends on a rather customized fork of `Formatting.jl`, i.e. `JuliaString/Format.jl`)

**Comment by jmkuhn** [comment]
Fixed with #88.

**Comment by ronisbr** [comment]
Hi @jmkuhn !

Is it possible to release a new version of Formatting.jl with this fix?
